packages/ts/form/src/BinderNode.ts

```diff
@@ -39,6 +40,30 @@ function getErrorPropertyName(valueError: ValueError<any>): string {
   return typeof valueError.property === 'string' ? valueError.property : getBinderNode(valueError.property).name;
 }
 
+/**
+ * Validator for verifyng that the `validity.valid` state is true. Used
+ * internally in the binder nodes to take the bound HTML element's internal
+ * validation state into account.
+ */
+export class ValidityStateValidator<T, M extends AbstractModel<T>> implements Validator<T> {
+  public message = '';
+
+  private binderNode: BinderNode<T, M>;
+
+  // eslint-disable-next-line no-useless-constructor
+  constructor(binderNode: BinderNode<T, M>) {
+    this.binderNode = binderNode;
+  }
+
+  validate(): boolean {
+    if (!this.binderNode[_validity]) {
+      return true;
```

Comment:
Codecov says that this line is not covered by tests. Are you going to add a test for this case?

Reply:
I ended up removing it. There's no point in this double checking of the state.